frontend/packages/text-editor/src/Variables.module.css

```diff
@@ -5,15 +5,34 @@
 
 .chip {
   border-radius: 20px;
-  width: 90%;
   margin: 5px;
   background-color: gray;
+  padding-left: 10px;
+  vertical-align: text-bottom;
+}
+
+.keyDataSource {
   display: inline-block;
   overflow: hidden;
+  width: 90%;
+  white-space: nowrap;
   text-overflow: ellipsis;
+  color: white;
+  font-size: 14px;
+}
+
+.defaultValue {
+  display: inline-block;
+  overflow: hidden;
+  width: 90%;
   white-space: nowrap;
+  text-overflow: ellipsis;
   color: white;
-  padding-left: 10px;
   font-size: 14px;
-  vertical-align: text-bottom;
 }
+
+.defaultValue::before {
+  content: 'Default: ';
```

Comment:
It's probably not so important for now, but one problem I see here is that we can't translate this text.
An easy fix could be to simply add another div or label with its own class, use the `useTranslation` hook, and move this text to `nb.json`.
I'm also wondering if it's ok to use `Default` in Norwegian, and if we should not use `Standard` or `Standardverdi` 🤔 🤷‍♂️